test/withRTLExtension_test.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import PropTypes from 'prop-types';
+import { expect } from 'chai';
+import * as aphrodite from 'aphrodite';
```

Comment:
why is this not `import aphrodite from 'aphrodite'`?

Reply:
I was following the pattern from here - https://github.com/airbnb/react-with-styles-interface-aphrodite/blob/master/src/aphroditeInterface.js#L1

Reply:
`import *` should be avoided; let's fix that in both places!